js/app/src/Index.svelte

```diff
@@ -166,6 +166,20 @@
 
 			if (parsed_head_html) {
 				for (let head_element of parsed_head_html) {
+					const head_element_property = head_element.getAttribute("property");
+
+					if (head_element.tagName == "META" && head_element_property) {
+						const domMetaList = Array.from(
+							document.head.getElementsByTagName("meta") ?? []
+						);
+						const matched = domMetaList.find(
+							(el) => el.getAttribute("property") == head_element_property
+						);
+						if (matched) {
+							document.head.removeChild(matched);
+						}
```

Comment:
If it matches (i.e. it is functionally the same meta tag) why remove it only to re-add it, can't we just skip the additional work and  `continue;` the loop here instead?

Reply:
My intention is to address the issue of duplication that arises when both `default meta tags are predefined in index.html` and `developers add meta tags` with the `head` parameter, which serve the same purpose. 

When rendered in the browser, two tags with similar functionalities exist, albeit with potentially different values or content. This duplication issue prevents the `meta tag specified by the developer` from functioning as intended.

I've written code to remove any existing `meta tags with equivalent functionality but differing values or content` before adding the specified HTML element with the additional `head` parameter. 

Additionally, I've provided further commits outlining improvements.
The improvement entails replacing `meta tags with the same property attribute`, ensuring that tags serving identical functions are substituted when they are not equivalent.

Thank you for your review.